gen-ai/orchestrator-server/src/main/python/server/src/gen_ai_orchestrator/services/langchain/rag_chain.py

```diff
@@ -148,6 +148,14 @@ async def execute_qa_chain(query: RagQuery, debug: bool) -> RagResponse:
         else None
     )
 
+def get_source_content(doc):
+    """Find and delete the title followed by two line breaks"""
+    title_prefix = f"{doc.metadata['title']}\n\n"
```

Comment:
Document that this pattern is used in the ingestion tooling script index_documents.py, in case someone wants to understand why we have that here or update the pattern.